### PR TITLE
Signer panel styling

### LINKED ISSUE #404: Signer Panel (scripts and contracts)
When the user is editing a script or contract:

 - only one signer can be selected

### Questions:

**How to handle if no signers are selected?**

One signer will be always selected by default.  As a rule, the first account` 0x01` will be the **default** selected account.   If a user selects another account, the previous selection will be disabled, that way only one account can be selected at a time:

Image

State change of the selected account (displayed on the top):

Image

How to handle if more than one signer is selected?
Do we display an error?

### Design
This design is generic and needs to above questions answered
https://www.figma.com/file/G7Z28scDlkNFvgDyR2nSnq/Playground-Library-%7C-Components-(External)?node-id=298%3A117977

image

### LINKED ISSUE #410: Signer Panel button and error count
Action button for signer panel.

 - [x] style button according to figma.
 - [x] enable and disable button accordingly
  
 Figma
https://www.figma.com/file/G7Z28scDlkNFvgDyR2nSnq/Playground-Library-%7C-Components-(External)?node-id=298%3A117977

### PR DESCRIPTION
**Pull request by bthaile**
Closes: #404 
Closes: #410 

 - format signer panel
 - style action button
 - show error count and hide error list
 - added types usage where they were missing
 - convert a few 'px' to use 'rem'
 - commented out Error list, will need to style it when the time comes. 
 
 Currently deploying contacts or running scripts fail, when that gets worked out I'll be able to finish "completed" button states.

massage redeploy confirmation dialog.
image

### CODE REVIEW

File: src/components/Arguments/index.tsx [hunk 1]
```diff
@@ -358,13 +357,13 @@ const Arguments: React.FC<ArgumentsProps> = (props) => {
   };
 
   const isOk = !haveErrors && validCode !== undefined && !!validCode;
-  let statusIcon = isOk ? <FaRegCheckCircle /> : <FaRegTimesCircle />;
-  let statusMessage = isOk ? 'Ready' : 'Fix errors';
+  //let statusIcon = isOk ? <FaRegCheckCircle /> : <FaRegTimesCircle />;
```
Comment (MrDSGC) [inline src/components/Arguments/index.tsx]: leftover commented code?

File: src/components/Arguments/index.tsx [hunk 2]
```diff
@@ -358,13 +357,13 @@ const Arguments: React.FC<ArgumentsProps> = (props) => {
...
+  let statusMessage = !isOk && `${problems?.error?.length} Errors`;
 
   const progress = isSaving || processingStatus;
 
   if (progress) {
-    statusIcon = <FaSpinner className="spin" />;
+    //statusIcon = <FaSpinner className="spin" />;
```
Comment (MrDSGC) [inline src/components/Arguments/index.tsx]: same as line 360

File: src/components/Arguments/index.tsx [hunk 3]
```diff
@@ -411,15 +410,17 @@ const Arguments: React.FC<ArgumentsProps> = (props) => {
             </>
           )}
 
-          <ErrorsList list={problems.error} actions={actions} />
+          {/*<ErrorsList list={problems.error} actions={actions} />*/}
```
Comment (MrDSGC) [inline src/components/Arguments/index.tsx:411]: same as line 360

File: src/components/CadenceEditor/ControlPanel/index.tsx [hunk 4]
```diff
@@ -398,54 +397,55 @@ const ControlPanel: React.FC<ControlPanelProps> = (props) => {
...
+          {needSigners && (
+            <SignersPanel
+              maxSelection={signers}
+              selected={selected}
+              updateSelectedAccounts={updateSelectedAccounts}
+            />
+          )}
+
+
+          {/*<ErrorsList list={problems.error} actions={actions} />*/}
```
Comment (MrDSGC) [inline src/components/CadenceEditor/ControlPanel/index.tsx:429]: commented out code?

File: src/components/PanelButton.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  isLoading?: boolean;
+  isComplete?: boolean;
+  isActive?: boolean;
+  disabled?: boolean;
+  hideDisabledState?: boolean;
+  'data-test'?: string;
+}
+
+const PanelButton: React.FC<PanelButtonProps> = (props) => {
+    // TODO: what is hideDisabledState used for
```
Comment (MrDSGC) [inline src/components/PanelButton.tsx:23]: indenting looks off

**Review by MrDSGC — COMMENTED** [review]
few minor comments then lgtm

File: src/components/CadenceEditor/ControlPanel/index.tsx (hunk 4 above)
Comment (bthaile) [reply src/components/CadenceEditor/ControlPanel/index.tsx:429]: leaving it commented for potential future work

File: src/components/Arguments/index.tsx (hunk 3 above)
Comment (bthaile) [reply src/components/Arguments/index.tsx:411]: leaving it commented for potential future work

File: src/components/PanelButton.tsx (hunk 5 above)
Comment (bthaile) [reply src/components/PanelButton.tsx:23]: formatter handles indentions using `yarn lint` command

File: src/components/Arguments/index.tsx (hunk 1 above)
Comment (bthaile) [reply src/components/Arguments/index.tsx]: addressed

File: src/components/Arguments/index.tsx (hunk 2 above)
Comment (bthaile) [reply src/components/Arguments/index.tsx]: addresses

**Review by MrDSGC — APPROVED** [review]
lgtm

**Pull request merged**